### PR TITLE
2.17 doc: Static Member Favorites

### PR DESCRIPTION
**Pull request by kthoms**
Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

File: xtend-website/_posts/releasenotes/2019-03-05-version-2-17-0.md [hunk 1]
```diff
@@ -51,10 +51,17 @@ If a class is named something like 'AbstractFoo' but is not declared abstract. A
 
 ## Static Favorites
 
-TODO Document me! I'm worth it!
+Annoyed by typing `org.junit.Assert.assertEquals` to use static methods? Wouldn’t it be cool having all your favourite static methods at hand and let the language do the import automatically? In JDT this feature is known as Static Member Favorites and it comes with a preference page, where you can configure which class’s methods should be visible. Besides wildcard imports you could make a single method visible - it’s up to you.
 
-![Use favorites for static import]({{site.baseurl}}/images/releasenotes/2_17_Xtend-static-favourites.gif){:height="50%" width="50%"}
+![Use favorites for static import]({{site.baseurl}}/images/releasenotes/2_17_Xtend-static-favorites_1.gif){:height="50%" width="50%"}
 
+We thought it’s a good idea to reuse the already defined preferences for Xtend. Since this release Xtend supports Static Member Favorites like Java does - but a bit more.
```
Comment (cdietrich) [inline xtend-website/_posts/releasenotes/2019-03-05-version-2-17-0.md]: JDT Preferences in Xtend?

Comment (kthoms) [reply xtend-website/_posts/releasenotes/2019-03-05-version-2-17-0.md]: Yes, the JDT preference is used. Same already for Type Filter.

Comment (cdietrich) [reply xtend-website/_posts/releasenotes/2019-03-05-version-2-17-0.md]: no i mean you should make this clean in the sentence.

Comment (cdietrich) [reply xtend-website/_posts/releasenotes/2019-03-05-version-2-17-0.md]: "We thought it’s a good idea to reuse the already defined JDT preferences in Xtend."

Comment (kthoms) [reply xtend-website/_posts/releasenotes/2019-03-05-version-2-17-0.md]: updated

**Pull request merged**